backoffice/templates/backoffice/home.html

```diff
@@ -20,6 +20,24 @@ <h1 class="text-xl font-semibold text-gray-900">Today and Tomorrow's bookings</h
         </div>
       </div>
     </div>
+    {% if late_bookings|length > 0 %}
+    <div class="sm:flex sm:items-center">
```

Comment:
Just tested this on my machine and the heading for this table is right up tight to the bottom of the table above. Can we add some padding or margin or something to put a bit of space between them for better visuals?

Reply:
Added the same gap as between Headings and tables.